src/preact/compat.js

```diff
@@ -24,3 +24,12 @@ import * as compat from /*OK*/ 'preact/compat';
 export function forwardRef(fn) {
   return compat.forwardRef(fn);
 }
+
+/**
+ * @param {PreactDef.VNode} vnode
+ * @param {Element} container
```

Comment:
I see the [Preact.createPortal](https://github.com/preactjs/preact/blob/7556b61bd0d9f72bbd906c3abaebf2b1c9aeacbf/compat/src/portals.js#L76) takes a `PreactElement` type for `container` which extends [`HTMLElement`](https://github.com/preactjs/preact/blob/7556b61bd0d9f72bbd906c3abaebf2b1c9aeacbf/src/internal.d.ts#L78). I don't think our `PreactDef` has anything like `PreactElement`, @jridgewell should we add this to our externs, use `HTMLElement`, or is this ok as is?

Reply:
`HTMLElement` is fine. They use `PreactElement` so they can attach private props to the element, but we don't need to know about that.